scipy/stats/tests/test_fit.py

```diff
@@ -10,7 +10,7 @@
 # this is not a proper statistical test for convergence, but only
 # verifies that the estimate and true values don't differ by too much
 
-fit_sizes = [1000, 5000]  # sample sizes to try
+fit_sizes = [1000, 5000, 10000]  # sample sizes to try
```

Comment:
Curious - why the addition of 10000?

Reply:
It is required to allow genhyperbolic to fit its parameters. See https://github.com/scipy/scipy/pull/13298/commits/a7ee3cde121ca8902061a97968a9076d6104200a . Can revert but will have to exclude genhyperbolic from this test.

Reply:
Hmm OK I'll have to look at that logic. I didn't know that it only uses the larger ones if needed.

Reply:
I see. OK, that's fine with me.